code/VintageEngineering/Transport/API/BEPipeBase.cs

```diff
@@ -452,9 +465,15 @@ public override void OnBlockBroken(IPlayer byPlayer = null)
         /// <param name="world">WorldAccessor object</param>
         public virtual void MarkPipeDirty(IWorldAccessor world, bool dirtyshape = false)
         {
+            if (Api == null) return; // I really don't like that fix: When we do a OnNeighbourBlockChange() for pipe in the chunk border it will call pipesBE that exist but not initialize yet ???
```

Comment:
"I really don't like that fix"
You don't like the fix you just added? 
If you don't like it, how am I supposed to like it?